priv/perf/apps/load_test/lib/watcher_info/utxo.ex

```diff
@@ -0,0 +1,101 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.w
+
+defmodule LoadTest.WatcherInfo.Utxo do
+  @moduledoc """
+  Functions for retrieving utxos through WatcherInfo API.
+  """
+  require Logger
+
+  alias LoadTest.Connection.WatcherInfo
+  alias LoadTest.Ethereum.Account
+  alias LoadTest.Service.Sync
+  alias LoadTest.Utils.Encoding
+
+  @poll_timeout 60_000
+
+  @spec get_utxos(Account.addr_t(), ExPlasma.Utxo.t() | nil | :empty) :: {:ok, [] | ExPlasma.Utxo.t()} | no_return
+  def get_utxos(sender, utxo \\ nil) do
+    Sync.repeat_until_success(
+      fn ->
+        fetch_utxos(sender, utxo)
+      end,
+      @poll_timeout,
+      "Failed to fetch utxos"
+    )
+  end
+
+  defp fetch_utxos(sender, utxo) do
+    case WatcherInfoAPI.Api.Account.account_get_utxos(
+           WatcherInfo.client(),
+           %WatcherInfoAPI.Model.AddressBodySchema1{
+             address: Encoding.to_hex(sender.addr)
+           }
+         ) do
+      {:ok, result} ->
+        result.body
+        |> Jason.decode!()
+        |> find_utxo(utxo)
+
+      other ->
+        other
+    end
```

Comment:
why not spend a little bit more time on the code design? who likes to read this, honestly?

```suggestion
client = WatcherInfo.client()
body = %WatcherInfoAPI.Model.AddressBodySchema1{address: Encoding.to_hex(sender.addr}
account_utxos = WatcherInfoAPI.Api.Account.account_get_utxos(client, body) 
    case account_utxos do
      {:ok, result} ->
        result.body
        |> Jason.decode!()
        |> find_utxo(utxo)

      other ->
        other
    end
```